src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -1105,4 +1172,10 @@ - (void)setPairingDelegate:(id<MTRDevicePairingDelegate>)delegate queue:(dispatc
     [self setDeviceControllerDelegate:delegateShim queue:queue];
 }
 
+- (void)setNocChainIssuer:(id<MTRNOCChainIssuer>)nocChainIssuer queue:(dispatch_queue_t)queue
+{
+    [self setOperationalCertificateIssuer:[[MTROperationalCertificateChainIssuerShim alloc] initWithIssuer:nocChainIssuer]
+                                    queue:queue
+                                    error:nil];
```

Comment:
isn't this dangerous if queue is nil and we try to set an error?

Reply:
Dangerous in what way?  The API contract is that error outparams are _always_ allowed to be nil; the implementation needs to handle that (and does).

Reply:
i actually missed the part where you were doing a check for the existence of the error var. My bad, please ignore!